config/request-docs.php

```diff
@@ -1,20 +1,21 @@
 <?php
 
 return [
-     // change it to true will make lrd to throw exception if rules in request class need to be changed
-     // keep it false
-    'debug'  => false,
-    'document_name'  => 'LRD',
+    // change it to true will make lrd to throw exception if rules in request class need to be changed
+    // keep it false
+    'debug' => false,
+    'document_name' => 'Fast Serve',
```

Comment:
Why do we need to change this?

Reply:
sorry i forget to change it again you are right we don't want this change